OpenOversight/app/templates/base.html

```diff
@@ -16,6 +16,13 @@
         OpenOversight - a Lucy Parsons Labs project
       {% endblock title %}
     </title>
+    {% if 'TIMEZONE' not in session %}
+      <script type="text/javascript">
+          // Send browser's timezone to server for displaying dates/times in local timezone.
+          const timezone = Intl.DateTimeFormat() ? Intl.DateTimeFormat().resolvedOptions().timeZone : "";
```

Comment:
<img width="1470" alt="Screenshot 2023-07-20 at 3 19 15 PM" src="https://github.com/lucyparsons/OpenOversight/assets/5885605/000a929c-3e9a-4e9b-b6b4-6f545ef8799e">

<img width="640" alt="Screenshot 2023-07-20 at 3 26 28 PM" src="https://github.com/lucyparsons/OpenOversight/assets/5885605/907e8b77-0b17-4da2-9351-7376d907db65">



We should be good to use this within the application as it is used within most browsers. 

Links:
- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/resolvedOptions
- https://caniuse.com/?search=Intl.DateTimeFormat().resolvedOptions().timeZone